guides/release/tutorial/8.md

```diff
@@ -0,0 +1,159 @@
+
+This is almost always the behavior you want, so you will almost always use `await` and `visit` as a pair. This applies to other kinds of simulated interaction too, such as clicking on a button or a link, as they all takes time to complete. Even though sometimes these actions may seem imperceptibly fast to us, we have to remember that our test robot has really really fast hands, as we will see in a moment.
+
+After navigating to the `/` URL and waiting for things to settle, we check that the current URL matches the URL that we expect (`/`). We can use the `currentURL` test helper here, as well as the `equal` *[assertion](TODO: link to assertion)*. This is how we encode our "checklist" into code &mdash; by specifying, or *[asserting](TODO: link to asserting)* how things should behave, we will be notified when our app does not behave in the way that we expect.
+
+Next, we confirmed that there should be a `<h1>` tag on the page that contains the text "Welcome to Super Rentals!". Knowing this is true means that we can be quite certain that the correct template is being rendered, without errors.
+
+Then, we looked for a link with the text `About Us`, located using the *[CSS selector](TODO: link to CSS selector)* `.jumbo a.button`. This is the same syntax we used in our stylesheet, which means "look inside the tag with the `jumbo` class, for an `<a>` tag with the `button` class". This matches up with the HTML structure in our template.
+
+Once the existence of this element on the page is confirmed, we told the test robot to click on this link. As mentioned above, this is a user interaction, so it needs to be `await`-ed.
+
+Finally, we asserted that clicking on the link should bring us to the `/about` URL.
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        Here, we are writing the tests in a framework called QUnit, which is where `module`, `test` and `assert` comes from. We also had additional helpers like `click`, `visit` and `currentURL` provided by the `@ember/test-helpers` package. You tell what came from which package based on the `import` paths at the top of the file. Knowing this will be helpful when you need to search for documentation on the Internet or ask for help.
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
+  </div>
+</div>
+
+We can put our automated test into motion by running the *[test server](TODO: link to test server)*, using the `ember test --server` command, or `ember t -s` for short. This server behaves like the development server, but it is explicitly running for our tests. It may automatically open a browser window and take you to the test UI, or you can open `http://localhost:7357/` yourself.
```

Comment:
```suggestion
We can put our automated test into motion by running the *[test server](TODO: link to test server)* using the `ember test --server` command, or `ember t -s` for short. This server behaves much like the development server, but it is explicitly running for our tests. It may automatically open a browser window and take you to the test UI, or you can open `http://localhost:7357/` yourself.
```